airflow/dag/serialization/serialization.py

```diff
@@ -148,7 +148,7 @@ def _serialize(cls, var):  # pylint: disable=too-many-return-statements
                 return cls._encode(var.isoformat(), type_=DAT.DATETIME)
             elif isinstance(var, datetime.timedelta):
                 return cls._encode(var.total_seconds(), type_=DAT.TIMEDELTA)
-            elif isinstance(var, pendulum.tz.Timezone):
+            elif isinstance(var, (pendulum.tz.Timezone, type(pendulum.UTC))):
```

Comment:
This looks suspect - what about using `datetime.tzinfo` instead, which both pendulum.tz.Timezone and pendulum.tz.timezone_info.TimezoneInfo inherit from.

Reply:
Thought about using that y'day but found an issue that Timezone and Timezoneinfo contains `name` property that we in the line below. datetime.tzinfo does not contain that

Reply:
Ah.

How about `pendulum.tz.timezone_info.TimezoneInfo` instead then - also `pendulum.UTC` is an instance, not the class itself. (The class is `pendulum.tz.timezone_info._UTC`)

Reply:
I did have this (`pendulum.tz.timezone_info.TimezoneInfo`)  first, however IDE complained (and so I guess mypy might have complained too - I guess not sure though): 

![image](https://user-images.githubusercontent.com/8811558/64258521-50ea9f00-cf1f-11e9-9919-92bef96ac908.png)

And so I changed it to ` type(pendulum.UTC)` which checks the type of the instance created.

Happy to try `pendulum.tz.timezone_info.TimezoneInfo` and see if mypy passes/fails